linux/just_makeself_functions.bsh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env false bash
+
+if [[ $- != *i* ]]; then
+  source_once &> /dev/null && return 0
+fi
+
+JUST_DEFAULTIFY_FUNCTIONS+=(makeself_defaultify)
+JUST_HELP_FILES+=("${BASH_SOURCE[0]}")
+
+source "${VSI_COMMON_DIR}/linux/dir_tools.bsh"
+
+#*# just/plugins/just_makeself_functions
+
+#**
+# ===========================
+# J.U.S.T. Makeself Functions
+# ===========================
+#
+# .. default-domain:: bash
+#
+# .. file:: just_makeself_functions.bsh
+#**
+
+#**
+# .. function:: makeself_defaultify
+#**
+function makeself_defaultify()
+{
+  arg=$1
+  shift 1
+
+  # Must be version after 2.4.2, I need a working ARCHIVE_DIR and append.
+  # https://github.com/megastep/makeself/issues/213
+  # https://github.com/megastep/makeself/issues/216
+  local JUST_MAKESELF_VERSION=${JUST_MAKESELF_VERSION:-d9a61e67803f95c4d91050932347811a00aa38e9}
+  local PROJECT_CWD="${JUST_PROJECT_PREFIX}_CWD"
+  PROJECT_CWD="${!PROJECT_CWD}"
+
+  case $arg in
+    #**
+    # .. command:: makeself_just
+    #
+    # Download and install makeself locally in Project's CWD/build/makeself
+    #**
+    makeself_setup-local) # Setup makeself locally
+      mkdir -p "${PROJECT_CWD}/build/makeself"
+      pushd "${PROJECT_CWD}/build/makeself"
+        curl -LO "https://github.com/megastep/makeself/archive/${JUST_MAKESELF_VERSION}/makeself.tar.gz"
+        tar xf makeself.tar.gz --strip-components=1
+        rm makeself.tar.gz
+
+        # Sisable makeself's argument parser (so all arguments go to just by
+        # default) and make quiet the default
+        sed '1,/^while true/s|^while true|while \\${MAKESELF_PARSE-false}|; 1,/^quiet="n"/s|^quiet="n"|quiet="y"|' \
```

Comment:
Should you give an example of what this sed does?

Reply:
I'm patching a file from makeself to make it more amenable to what I want, what do you mean give an example?